bin/package.js

```diff
@@ -508,6 +506,11 @@ function buildLinux (cb) {
     // Create .deb file for Debian-based platforms
     console.log(`Linux: Creating ${destArch} deb...`)
 
+    // Linux convention for Debian based 'x64' is 'amd64'
+    if (destArch === 'x64') {
+      destArch = 'amd64'
+    }
```

Comment:
Nit: Can you move this line above the `console.log` above so that the `Linux: Creating ${destArch} deb...` and `Linux: Created ${destArch} deb.` print statements will refer to the architecture using the same name? ;)

Reply:
Sure!